### DISCUSSION TITLE
Dialog got margin, when scroll and open dialog

### DISCUSSION THREAD

**Discussion by xxidbr9** [tailwindlabs/headlessui#2124] [Help]
I got an issue when I have a ```position: fixed```  component inside the body, the body got (-margin) each scroll and opened the dialog
```js
style(ownerDocument.body, 'marginTop', `-${scrollPosition}px`)
```
from: https://github.com/tailwindlabs/headlessui/blob/main/packages/%40headlessui-react/src/components/dialog/dialog.tsx#L126-L156

Could anyone explain to me?

**Comment by Kovrigin-Al** [comment]
I have the same issue. In my case, this is even worse as the scrollIntoView method does not work correctly because of this.
Hopefully, some mature developer will come to rescue :)

**Reply by Kovrigin-Al** [reply]
Update: Popover works for me. It doesn't add negative margin
